### PR TITLE
Add jupyterlab-tour

### PR DESCRIPTION
**Pull request by fcollonval**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] ~~If static libraries are linked in, the license of the static library is packaged.~~
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by fcollonval** [comment]
Thanks for the [reminder](https://github.com/jupyterlab-contrib/jupyterlab-tour/issues/10) @bollwyvl 

I added you as maintainer - let me know if it is ok for you?

**Comment by bollwyvl** [comment]
> I added you as maintainer - let me know if it is ok for you?
>
Ok!

>

**Comment by fcollonval** [comment]
@conda-forge/help-python this is ready for review

**Comment by bollwyvl** [comment]
Also sometimes pinging @staged-recipes works, too, being more targeted.

File: recipes/jupyterlab-tour/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  sha256: a1fb867d5278f27dc43ffd8a976788e5a6fff769c7f482aed5f6c9aace9b47a1
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - nodejs >=12.0.0
```
Comment (bollwyvl) [inline recipes/jupyterlab-tour/meta.yaml:19]: Actually, if there's a way to use the assets directly from the sdist (e.g. by deleting the `pyproject.toml`) that can reduce the chances of getting... surprises.

File: recipes/jupyterlab-tour/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  run:
+    - python >=3.6
+    - jupyterlab >=3.0.0
+
+test:
+  requires:
+    - nodejs >=12.0.0
+  commands:
+    - jupyter labextension list 2>&1 | grep -ie "jupyterlab-tour.*OK"
+    - python -m jupyterlab.browser_check  # [not linux]
```
Comment (bollwyvl) [inline recipes/jupyterlab-tour/meta.yaml:34]: i don't think browser_check adds much value, and can break for weird reasons unrelated to the system-under-build (e.g. build container doesn't have system x11 drivers, etc). If we're shipping the stuff directly from the sdist, just going with the labextension check is probably sufficient, and more likely to catch packaging errors vs nodejs/browser/driver errors.

```yaml
        - jupyter labextension list
        - jupyter labextension list 1>labextensions 2>&1
        - cat labextensions | grep "jupyterlab-tour.*OK.*jupyterlab_tour"  # [unix]
```

File: recipes/jupyterlab-tour/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+about:
+  home: https://github.com/jupyterlab-contrib/jupyterlab-tour
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE
+  summary: 'A JupyterLab UI tour built on jupyterlab-tutorial and react-joyride.'
+  dev_url: https://github.com/jupyterlab-contrib/jupyterlab-tour
```
Comment (bollwyvl) [inline recipes/jupyterlab-tour/meta.yaml:42]: when duplicating `home`, this doesn't provide much additional insight...

File: recipes/jupyterlab-tour/meta.yaml (hunk 1 above)
Comment (bollwyvl) [reply recipes/jupyterlab-tour/meta.yaml:19]: skip-npm might suficient:

```
{{ PYTHON }} -m pip install --install-option="--skip-npm" . -vv
```

Comment (fcollonval) [reply recipes/jupyterlab-tour/meta.yaml]: Thanks for this great suggestion

**Comment by fcollonval** [comment]
Thanks @bollwyvl for the review.

**Comment by bollwyvl** [comment]
@conda-forge/staged-recipes ready for review again!

**Pull request merged**